docs/grpc/c#.md

```diff
@@ -13,67 +13,39 @@ This section enumerates what you need to do to write a client that communicates
-```
+Create a folder `Grpc` in the root of your project and fetch the lnd proto files
 
-* Create the necessary folder structure, and then fetch the lnd [rpc.proto](https://github.com/lightningnetwork/lnd/blob/master/lnrpc/rpc.proto) file:
 ```bash
 mkdir Grpc
 curl -o Grpc/rpc.proto -s https://raw.githubusercontent.com/lightningnetwork/lnd/master/lnrpc/rpc.proto
 ```
 
-* Copy Google's [annotations.proto](https://github.com/googleapis/googleapis/blob/master/google/api/annotations.proto) to the correct folder:
-```bash
-mkdir Grpc/google
-mkdir Grpc/google/api
-curl -o Grpc/google/api/annotations.proto -s https://raw.githubusercontent.com/googleapis/googleapis/master/google/api/annotations.proto
-```
+Install `Grpc.Tools`, `Google.Protobuf`, `Grpc.Core` using NuGet or manually with `dotnet add`:
 
-* Copy Google's [http.proto](https://github.com/googleapis/googleapis/blob/master/google/api/http.proto) to the correct folder:
 ```bash
-curl -o Grpc/google/api/http.proto -s https://raw.githubusercontent.com/googleapis/googleapis/master/google/api/http.proto
+dotnet add package Grpc.Tools
+dotnet add package Google.Protobuf
+dotnet add package Grpc.Core
 ```
 
-* Copy Google's [descriptor.proto](https://github.com/protocolbuffers/protobuf/blob/master/src/google/protobuf/descriptor.proto) to the correct folder:
-```bash
-mkdir Grpc/google/protobuf
-curl -o Grpc/google/protobuf/descriptor.proto -s https://raw.githubusercontent.com/protocolbuffers/protobuf/master/src/google/protobuf/descriptor.proto
-```
+Add the `rpc.proto` file to the `.csproj` file in an ItemGroup. (In Visual Studio you can do this by unloading the project, editing the `.csproj` file and then reloading it)
 
-* Compile the proto file using `protoc.exe` from nuget package `Grpc.Tools` (possibly replace "YOUR_USER", version "1.17.0", or your OS in both paths):
-```bash
-# linux + mac nuget package location: ~/.nuget/packages
-cd Grpc
-C:/Users/<YOUR_USER>/.nuget/packages/grpc.tools/1.17.0/tools/windows_x64/protoc.exe --csharp_out . --grpc_out . rpc.proto --plugin=protoc-gen-grpc=C:/Users/<YOUR_USER>/.nuget/packages/grpc.tools/1.17.0/tools/windows_x64/grpc_csharp_plugin.exe
+```
+<ItemGroup>
+   <Protobuf Include="Grpc\rpc.proto" GrpcServices="Client" />
+</ItemGroup>
 ```
 
-
-After following these steps, two files `Rpc.cs` and `RpcGrpc.cs` will be generated in the `Grpc` folder in your project.
-
-
+You're done! Build the project and verify that it works.
 
 #### Imports and Client
 
-Every time you use C# `gRPC`, you will have to import the generated rpc classes, and use `nuget` package manger to install `Grpc.Core` (1.17.0 at time of writing), `Google.Protobuf` (3.6.1), and `Google.Api.CommonProtos` (1.4.0).
-
-```bash
-# from project root, install packages using nuget 
-cd ../
-dotnet add package Grpc.Core --version 1.17.0
-dotnet add package Google.Protobuf --version 3.6.1
-dotnet add package Google.Api.CommonProtos --version 1.4.0
-```
-
```

Comment:
The text below this probably needs some adjusting, because "After installing these" is no longer accurate.

Reply:
Thanks for the comment. Fixed.

Reply:
Yes, the setup is tested.